bootstrap/kubeadm/config/default/kustomization.yaml

```diff
@@ -23,6 +24,27 @@ bases:
 # [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
 #- ../prometheus
 
+patchesJson6902:
```

Comment:
Only to double check (I'm not a kustomize expert):
Patches Json6902 are applied after the commonLabels, doesn't it?

Reply:
I will confirm but as part of my testing it seems to work as intended.

Reply:
I asked in slack and they pointed me to this: https://github.com/kubernetes-sigs/kustomize/blob/c7d78970fb86782dbdded3a93944b774f826071f/api/internal/target/kusttarget_configplugin.go#L44

PatchJson6902 transformer is applied after LabelTransformer